docs/reference/method/MongoDBCollection__construct.txt

```diff
@@ -19,15 +19,65 @@ Definition
 
    .. code-block:: php
 
-      function __construct(MongoDB\Driver\Manager $manager, string $databaseName, string $collectionName, array $options = [])
+      function __construct(
+          MongoDB\Driver\Manager $manager,
+          string $databaseName,
+          string $collectionName,
+          array $options = []
+      )
 
    This constructor has the following parameters:
 
-   .. include:: /includes/apiargs/MongoDBCollection-method-construct-param.rst
+``$manager`` : :php:`MongoDB\\Driver\\Manager <class.mongodb-driver-manager>`
+  The :php:`Manager <mongodb-driver-manager>` instance from the driver. The
+  manager maintains connections between the driver and your MongoDB instances.
 
-   The ``$options`` parameter supports the following options:
+``$databaseName`` : string
+  The name of the database.
 
-   .. include:: /includes/apiargs/MongoDBCollection-method-construct-option.rst
+``$collectionName`` : string
+  The name of the collection.
+
+``$options`` : array
+  An array specifying the desired options.
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 20 20 80
+
+     * - Name
+       - Type
+       - Description
+
+     * - readConcern
```

Comment:
The `codec` option is missing.

Reply:
Defer to @alcaeus here, but I think this was intentionally skipped in [PHPLIB-1182](https://jira.mongodb.org/browse/PHPLIB-1182) since we wanted to roll out the feature quietly. I don't think there are any tickets to track that, so I just created [PHPLIB-1322](https://jira.mongodb.org/browse/PHPLIB-1322).